### PR TITLE
Removed binary from repo

### PR DESCRIPTION
**Pull request by trentrosenbaum**

### CODE REVIEW

**Review by matpimenta — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -17,3 +20,8 @@ jobs:
       - name: Check outputs
         run: |
           test "${{ steps.selftest.outputs.myOutput }}" == "Hello Golang"
+  approved:
+    if: github.event.review.state == 'approved'
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "This PR was approved"
```
Comment (ghost) [inline .github/workflows/integration.yml]: Looking great :)

**Review by ghost — CHANGES_REQUESTED** [review]
Please make some updates

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Needs changes

**Review by ghost — APPROVED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
more changes

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
not good enough

**Review by ghost — DISMISSED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
lol

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
lol

**Pull request closed without merging**